pkg/config/config.go

```diff
@@ -324,22 +315,35 @@ func parseMode(raw string) (bool, error) {
 	return raw == "init", nil
 }
 
-func parseTargetDomainRoles(raw string) ([]DomainRole, error) {
+// parseTargetDomainRoles parses a raw string containing domain and role pairs
+// separated by commas. If the input string does not contain ":role",
+// the entire string is considered as the domain and the role is set to an empty string.
+// All successfully split pairs are stored in the domainRoles slice.
+func parseTargetDomainRoles(raw string) ([]DomainRole, []DomainRole) {
 	elements := strings.Split(raw, ",")
-	errs := make([]error, 0, len(elements))
-	domainRoles := make([]DomainRole, 0, len(elements))
+	RoleCertDomainRoles := make([]DomainRole, 0, len(elements))
+	TokenDomainRoles := make([]DomainRole, 0, len(elements))
```

Comment:
```suggestion
	roleCertDomainRoles := make([]DomainRole, 0, len(elements))
	tokenDomainRoles := make([]DomainRole, 0, len(elements))
```

Reply:
fixed in: https://github.com/AthenZ/k8s-athenz-sia/pull/136/commits/10af4d896c5e1c7ba92fc554bf36ac27b7d54f29